### PR TITLE
Document web dashboard

### PR DESCRIPTION
**Pull request by keirthana**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 2 | howto/manage/web-dashboard | [Use the web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871)|
 | 2 | howto/application/landing | [Manage applications](https://discourse.ubuntu.com/t/manage-applications/24333) |
 | 3 | howto/application/create | [Create an application](https://discourse.ubuntu.com/t/create-an-application/24198)|
+| 3 | howto/application/stream | [Stream an application](tbd)|
```
Comment (jat-canonical) [inline index.md:70]: Would this be updated after you post the stream.md on discourse ?

**Review by jat-canonical — APPROVED** [review]
Just small question. Otherwise LGTM

File: index.md (hunk 1 above)
Comment (keirthana) [reply index.md:70]: yes, for new topics, we get the link after we post to discourse and then update it.

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**